core/src/com/projectkorra/projectkorra/event/PlayerChangeElementEvent.java

```diff
@@ -74,9 +74,9 @@ public Result getResult() {
 		return this.result;
 	}
 
-	public static enum Result {
-		CHOOSE, REMOVE, ADD, PERMAREMOVE;
-		private Result() {}
+	public enum Result {
+		CHOOSE, REMOVE, ADD, PERMAREMOVE,
+		TEMP_ADD, TEMP_REMOVE, TEMP_EXPIRE;
```

Comment:
May want to fire a `PlayerChangeElementEvent` or `PlayerChangeSubElementEvent` event in `TempCommand.java`, so developers can actually track temp changes when listened for, like around here:
https://github.com/ProjectKorra/ProjectKorra/blob/551c6bbba0a750c068c3c94ffe201fe195570366/core/src/com/projectkorra/projectkorra/command/TempCommand.java#L174-L188

or within the `addElement`, `extendElement`, `reduceElement`, `removeElement` functions.

Reply:
Possibly an oversight